doc/source/reference/series.rst

```diff
@@ -76,8 +76,8 @@ Indexing, iteration
    Series.loc
    Series.iloc
    Series.__iter__
-   Series.iteritems
    Series.items
+   Series.iteritems
```

Comment:
did you mean to add this back?

Reply:
Yes, as we’re not deprecating itertools anymore?